src/main/webapp/placeGuides/PlaceGuideOnMap.js

```diff
@@ -102,40 +97,13 @@ class PlaceGuideOnMap {
     this._marker.setMap(null);
   }
 
-  highlightOnMarkerDoubleClick() {
-    const thisPlaceGuideOnMap = this;
-    this._marker.addListener('dblclick', () => {
-      if (!thisPlaceGuideOnMap.isHighlighted()) {
-        placeGuideManager.highlightPlaceGuide(thisPlaceGuideOnMap.id);
-      }
-    });
-  }
-
-  toggleInfoWindowOnMarkerClick() {
-    const thisPlaceGuideOnMap = this;
-    this._marker.addListener('click', () => {
-      if (thisPlaceGuideOnMap._infoWindowClosed) {
-        thisPlaceGuideOnMap.openInfoWindow();
-      } else {
-        thisPlaceGuideOnMap.closeInfoWindow();
-      }
-    });
-  }
-
-  unhighlightOnMarkerClick() {
+  toggleHighlightOnMarkerClick() {
     const thisPlaceGuideOnMap = this;
     this._marker.addListener('click', () => {
       if (thisPlaceGuideOnMap.isHighlighted()) {
-        placeGuideManager.unhighlightPlaceGuide();
-      }
-    });
-  }
-
-  closeInfoWindowOnMapClick() {
-    const thisPlaceGuideOnMap = this;
-    map.addListener('click', function(mapsMouseEvent) {
-      if (!thisPlaceGuideOnMap._infoWindowClosed) {
-        thisPlaceGuideOnMap.closeInfoWindow();
+        placeGuideManager.unhighlightPlaceGuide(thisPlaceGuideOnMap.id);
+      } else {
+        placeGuideManager.highlightPlaceGuide(thisPlaceGuideOnMap.id);
```

Comment:
I don't see a change in this PR to open the info window. Was this always part of highlightPlaceGuide or is it in another PR?

Reply:
It was already part of the highlight() function of PlaceGuideOnMap. I only removed the animation.